package.json

```diff
@@ -31,6 +31,7 @@
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.17.2",
     "expect.js": "0.3.x",
+    "glob": "^7.1.6",
```

Comment:
What does that do?

Reply:
We use this library to dynamically import all the assertions from the ./assertions/ directory (instead of adding them one by one), glob gives you the glob syntax for doing that